swfo/brdf_fallback.py

```diff
@@ -400,34 +427,104 @@ def __get_data(dat_filename, param, window):
     return np.nanstd(data, axis=0, ddof=1, keepdims=False)
 
 
-def concatenate_files(infile_paths, outfile, h5_info, doy):
+def _calculate_valid_bounds(mask: np.Array, transform: Affine) -> Dict:
```

Comment:
change input variable to reflect numpy array (mask: numpy.array)

Reply:
thanks :)